index.html

```diff
@@ -5477,30 +5483,30 @@ <h3>Algorithm</h3>
                   <code>xsd:double</code> and its
                   <a>lexical form</a>
                   is a valid <code>xsd:integer</code> or <code>xsd:double</code>
-                  according to [[XMLSCHEMA11-2]]:
-                  <ol>
-                    <li>
-                      <ins>
+                  according to [[XMLSCHEMA11-2]]<del cite="#change_5">,
+                    set <var>converted value</var>
+                    to the result of converting the
+                    <a>lexical form</a>
+                    to a JSON <a>number</a>.
+                  </del><ins cite="#change_5">:
```

Comment:
:thinking: I thought `<ins>` cannot wrap an `<ol>`, can it?

Reply:
The validator does not flag it, and I don't know why it shouldn't be able to wrap `ol`.